cli/src/main/resources/help/help.txt

```diff
@@ -77,7 +82,7 @@ Usage:
                    to return. The value provided must be a positive integer.
                    Defaults to 5000 milliseconds when not provided.
 
-  --bind         - specifies to which address the CLI is going to be bound to.
+ --bind          - specifies to which address the CLI is going to be bound to.
```

Comment:
is going to be bound.

No final "to" is needed as the clause begins with "to which".